app/core/migrations/0004_auto_20220610_1000.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.13 on 2022-06-10 10:00
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0003_auto_20220609_1308'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Ingredient',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='recipe',
+            name='ingredients',
+            field=models.ManyToManyField(to='core.Ingredient'),
```

Comment:
Why this? Based on the requirements : 
_assume a given ingredient belongs only to one recipe, even if that means multiple Ingredient instances with the exact same name._